Source/CarthageKit/VersionFile.swift

```diff
@@ -79,13 +79,13 @@ struct VersionFile: Codable {
 		self = versionFile
 	}
 
-    static func url(for dependency: Dependency, rootDirectoryURL: URL) -> URL {
-        let rootBinariesURL = rootDirectoryURL
+	static func url(for dependency: Dependency, rootDirectoryURL: URL) -> URL {
+		let rootBinariesURL = rootDirectoryURL
 			.appendingPathComponent(Constants.binariesFolderPath, isDirectory: true)
 			.resolvingSymlinksInPath()
-        return rootBinariesURL
+		return rootBinariesURL
 			.appendingPathComponent(".\(dependency.name).\(VersionFile.pathExtension)")
-    }
+	}
```

Comment:
This change is unrelated — I saw some un-tabbed indentation and reindented it. Happy to drop in the name of brevity, though 🙂